insights/parsers/numeric_user_group_name.py

```diff
@@ -0,0 +1,62 @@
+"""
+NumericUserGroupName - command ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+=========================================================================================
+
+This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
+and detects whether it is 0 or nonzero.
+
```

Comment:
Explaining possible use of this parser would be useful. Please add here anything from the related bug report.